net-p2p/bitcoin-core/bitcoin-core-25.1-r1.ebuild

```diff
@@ -254,6 +305,14 @@ pkg_postinst() {
 			- Running bitcoind directly: add that user to the 'tor' user group.
 			EOF
 	fi
+
+	if use bitcoin-cli && use daemon ; then
+		efmt -su elog <<-EOF
+			To use bitcoin-cli with the /etc/init.d/bitcoind service:
```

Comment:
```suggestion
			To use bitcoin-cli with the ${EROOT}/etc/init.d/bitcoind service:
```

Reply:
You also can't start/use a service that's installed to a prefix, so `${EPREFIX}` here would also be wrong.

Reply:
I take the point, but for me, the question is whether or not someone will want to configure something and being able to copy/paste it is worth it in the messages.

Reply:
Why would anyone be copy/pasting `${EROOT}/etc/init.d/bitcoind`? You can't do anything useful with that string. Actually the _more_ useful string omits `${EROOT}` since you might be starting `/etc/init.d/bitcoind` in a chroot (although really you should be calling `rc-service bitcoind start` for that anyway).

Reply:
One might want to inspect the init script.